src/pages/IndexPage.vue

```diff
@@ -33,6 +33,11 @@
         <q-banner class="text-center">
           <div class="text-bold">Requires at least 4 participants</div>
           <div class="text-h6">Scan to join</div>
+          <div class = "text=center">
```

Comment:
its common practice to remove space between property and value see suggestion, 
  ```suggestion
          <div class="text-center">
```
text-center is using dash instead of =